scipy/conftest.py

```diff
@@ -5,6 +5,7 @@
 
 from distutils.version import LooseVersion
 import numpy as np
+import numpy.testing as npt
```

Comment:
nit: `import numpy.typing as npt` is also used in the NumPy official docs: https://numpy.org/devdocs/reference/typing.html#number-precision

We're unlikely to import that in `conftest.py` anyway, so I don't really care much I don't think. I've seen this in other code bases too--perhaps any confusion on the standard alias will clear up over time.

Reply:
I don't mind either way, this abbreviation or just `from numpy.testing import what_one_needs`